### ISSUE TITLE
It's February 10 and no release

### ISSUE THREAD

**Issue by Sultron** [vibora-io/vibora#235]
Project is said to be re-written and released on February 10th. What happened?

**Comment by yxxhero** [comment]
Soft fire makes sweet malt

**Comment by frnkvieira** [comment]
Hi @Sultron,
Sorry to disappoint you but I'm late :P
Feel free to reach me on Slack if you want help with something or just ask me about the progress.
I appreciate your interest, thanks!

**Comment by Team-OF** [comment]
@frnkvieira
Hey : )
we are still watining , please release it as soon as possible 
Thanks

**Comment by shrike71** [comment]
Hey Frank,

No worries... always better to get it right before getting it sooner. However, can you update the release date if it'll be pushed out much further?

Thanks

**Comment by keshon** [comment]
> Hi @Sultron,
> Sorry to disappoint you but I'm late :P
> Feel free to reach me on Slack if you want help with something or just ask me about the progress.
> I appreciate your interest, thanks!

Yeah I also would like to know at least some raw guess about a release date. Could you please post some info here without a Slack? Thank you!

**Comment by RealLau** [comment]
> Hi @Sultron,
> Sorry to disappoint you but I'm late :P
> Feel free to reach me on Slack if you want help with something or just ask me about the progress.
> I appreciate your interest, thanks!

@frnkvieira can u share the slack channel? So others can join in. I'm a tester, and I want do some help like some tests or other things...before next release if  possible.

**Comment by supposedly** [comment]
@RealLau it is linked to in the "join | slack" README badge, although I'll agree it's easy to miss. [Link here](https://join.slack.com/t/vibora-io/shared_invite/enQtNDAxMTQ4NDc5NDYzLTA2YTdmNmM0YmY4ZTY0Y2JjZjc0ODgwMmJjY2I0MmVkODFiYzc4YjM0NGMyOTkxMjZlNTliZDU1ZmFhYWZmNjU)

Seconding the request for communication outside of Slack however

**Comment by RealLau** [comment]
@supposedly Thanks for sharing, I think I'm in now. 👌
